glean_parser/templates/kotlin.jinja2

```diff
@@ -37,6 +37,7 @@ import {{ glean_namespace }}.private.LabeledMetricType // ktlint-disable import-
 internal object {{ category_name|Camelize }} {
 {% for obj in objs.values() %}
     {% if obj.extra_keys|length %}
+    @Suppress("ClassNaming", "EnumNaming")
```

Comment:
Just suppressing these now because fixing would require a change in the public API.

Reply:
Do we have a bug for fixing this? If so, can you mention it as a comment in here?

Reply:
I thought we could just keep [this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1590080) to track that.  I don't consider this PR as fixing that bug, just making it easier to address that bug properly when we decide how to proceed (and preventing bugs of that type in the future)...